contentcuration/contentcuration/serializers.py

```diff
@@ -220,12 +220,10 @@ class ContentNodeSerializer(BulkSerializerMixin, serializers.ModelSerializer):
     original_channel = serializers.SerializerMethodField('retrieve_original_channel')
 
     def retrieve_original_channel(self, node):
-        if node.original_node is None:
-            return None
-        root = node.original_node.get_root()
-        root_channel = root.channel_main or root.channel_trash or root.channel_clipboard or root.channel_staging or root.channel_previous
-        if root_channel.first():
-            return {"id": root_channel.first().pk, "name": root_channel.first().name}
+        if node.original_node:
+            root_channel = node.original_node.get_channel()
```

Comment:
You may want to add a TODO to delete/update this once we've updated/transferred everything, if I'm understanding this correctly?